cmd/bridge/main.go

```diff
@@ -418,6 +429,15 @@ func main() {
 
 		srv.TerminalProxyTLSConfig = serviceProxyTLSConfig
 
+		if *fK8sModeOffGitOps != "" {
```

Comment:
![image](https://user-images.githubusercontent.com/20724543/87326365-42faa700-c550-11ea-9276-b7d36a71c22a.png)

@chetan-rns it seems you are storing the route as an env variable but haven't removed the flag yet and somehow that is messing up the URL.

Reply:
`Error from server (NotFound): namespaces "pipelines-app-delivery" not found`
The error is after running the source command. The env variable is not created because the namespace is not found. Please check if the operator is installed

Reply:
Okay so I started the operator installation and it failed for some reason. I have again installed it was successful this time and I can use the proxy now. But now it fails to fetch manifest.
![image](https://user-images.githubusercontent.com/20724543/87331881-5742a200-c558-11ea-9813-a6e7e5f974a9.png)

Reply:
So I see this https://github.com/bigkevmcd/gitops.git doesnt work anymore

Reply:
I can grant access to it.

We should be testing _only with a private_ repo, because if it works with private repos, then it will work with public repos.